### PR TITLE
Gradle: Rework code slight to get rid of return@

### PR DESCRIPTION
**Pull request by sschuberth**
The return@ syntax might be hard to read.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/45)

### CODE REVIEW

**Comment by sschuberth** [comment]
@mnonnenmacher, this is just an idea. See yourself whether you like it!

File: analyzer/src/main/kotlin/managers/Gradle.kt [hunk 1]
```diff
@@ -146,45 +146,45 @@ class Gradle : PackageManager() {
             // Only look for a package when there was no error resolving the dependency.
             packages.getOrPut("${dependency.groupId}:${dependency.artifactId}:${dependency.version}") {
                 if (dependency.pomFile.isNotBlank()) {
-                    val projectBuildingResult = maven.buildMavenProject(File(dependency.pomFile))
-                    projectBuildingResult.project.let {
-                        return@getOrPut Package(
+                    with(maven.buildMavenProject(File(dependency.pomFile))) {
```
Comment (mnonnenmacher) [inline analyzer/src/main/kotlin/managers/Gradle.kt]: Isn't a "project ->" missing at the end of the line? I just wonder where the name "project" comes from below.

Comment (mnonnenmacher) [reply analyzer/src/main/kotlin/managers/Gradle.kt]: Ok, I found out myself.

**Comment by mnonnenmacher** [comment]
Nit: Should be "slightly" in the commit message.

**Comment by sschuberth** [comment]
> Nit: Should be "slightly" in the commit message.

Done.

**Pull request merged**